test/specs/RangeHighlightsSpec.js

```diff
@@ -38,6 +40,39 @@ describe("RangeHighlights Render Tests", function() {
                 { "start": 23, "end": 15 }  //out of range - not visible
             ]
         });
+
+        testSlider3 = $('#testSlider3').slider({
```

Comment:
could you properly tear down `testSlider3` in the `afterEach` block (lines 79 - 85)?